CoreArrangementPlugins/release/modules/ext/docs/CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/flatten-z-field.md

```diff
@@ -14,11 +14,24 @@
 <td>Run Flatten Z-Field</td>
 <td></td>
 <td>Arrange -&gt; Flatten Z-Field</td>
-<td style="text-align: center;"><img src="../ext/docs/CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/resources/zflat.png" alt="Flatten Z-Field Icon" /></td>
+<td style="text-align: center;"><img src="../ext/docs/CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/resources/ZFlatIcon.png" alt="Flatten Z-Field Icon" /></td>
 </tr>
 </tbody>
 </table>
 
-Flatten Z-Field sets the Z attribute of all nodes to zero, making the
-graph 2D. This is performed on all nodes, regardless of the current
-selection.
+Flatten Z-Field sets the Z attribute of all selected nodes to zero, making the
+graph 2D. If no nodes are selected, then all nodes of the graph will be flattened.
+
+
+Example Flatten:
+
+<div style="text-align: center">
+    <figure style = "display: inline-block">
+        <img height=400 src="../ext/docs/CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/resources/BeforeZFlat.png" alt="Before Flatten" />
+        <figcaption>Before Flatten</figcaption>
+    </figure>
+    <figure style = "display: inline-block">
+        <img height=400 src="../ext/docs/CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/resources/ZFlat.png" alt="After Flatten" />
+        <figcaption>After Flatten</figcaption>
+    </figure>
+</div>
```

Comment:
I'm not sure these images demonstrate the flattening particularly well as  its not clear that the flattening has occurred. For a better demonstration, it might be worth taking screenshots from a different camera angle (so that you can see graph the effects across the z-axis